packages/notifi-dapp-example/src/context/GlobalStateContext.tsx

```diff
@@ -13,25 +13,33 @@ import {
   useState,
 } from 'react';
 
+type IsInitialized = 'notInitialize' | 'initialized' | 'initializing';
+
 export type GlobalStateContextType = {
   isGlobalLoading: boolean;
   setIsGlobalLoading: Dispatch<SetStateAction<boolean>>;
   globalError: null | string;
   setGlobalError: Dispatch<SetStateAction<string | null>>;
+  isInitialized: IsInitialized;
+  seIisInitialized: Dispatch<SetStateAction<IsInitialized>>;
 };
 
 const GlobalStateContext = createContext<GlobalStateContextType>({
   isGlobalLoading: false,
   setIsGlobalLoading: () => undefined,
   globalError: null,
   setGlobalError: () => undefined,
+  isInitialized: 'notInitialize',
```

Comment:
Hmmm ... I would not suggest to put notifi related code into the "globalStateContext" since it is meant to used for global wised.